### PR TITLE
fluxus: init at 0.19

### PR DESCRIPTION
**Pull request by brainrake**
###### Motivation for this change

Great piece of software, wouldn't want to keep the package definition to myself.
http://www.pawfal.org/fluxus/

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/fluxus/default.nix [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    liblo
+    libpng
+    libsndfile
+    libtiff
+    ode
+    openal
+    openssl
+    zlib
+  ];
+  libPath = stdenv.lib.concatStringsSep " " (map (l: l.out + "/lib") libs);
```
Comment (marsam) [inline pkgs/applications/graphics/fluxus/default.nix]:
```suggestion
  libPath = stdenv.lib.makeLibraryPath libs;
```

File: pkgs/applications/graphics/fluxus/default.nix [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+in
+stdenv.mkDerivation rec {
+  name = "fluxus-${version}";
```
Comment (marsam) [inline pkgs/applications/graphics/fluxus/default.nix]:
```suggestion
  pname = "fluxus";
```

File: pkgs/applications/graphics/fluxus/default.nix [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    libjack2
+    libjpeg.dev
+    liblo
+    libsndfile.dev
+    libtiff.dev
+    ode
+    openal
+    openssl.dev
+    racket
+    scons
```
Comment (marsam) [inline pkgs/applications/graphics/fluxus/default.nix]: please move `scons` to `nativeBuildInputs`

File: pkgs/applications/graphics/fluxus/default.nix [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  ];
+  patches = [ ./fix-build.patch ];
+  buildPhase = ''
+    scons \
+      Prefix="$out" \
+      RacketPrefix="${racket}" \
+      RacketInclude="${racket}/include/racket" \
+      RacketLib="${racket}/lib/racket" \
+      LIBPATH="${libPath}" \
```
Comment (marsam) [inline pkgs/applications/graphics/fluxus/default.nix]: can you use `sconsFlags` for these?

**Comment by brainrape** [comment]
Thank you for the review!

**Comment by marsam** [comment]
@brainrape thank you for packaging this

**Pull request merged**